app/Models/Image.php

```diff
@@ -154,6 +125,37 @@ public static function upload(UploadedFile $file, $user, bool $forceReupload = f
         }
     }
 
+    /**
+     * Converts the image into the specified cover type to the specified path.
+     *
+     * @param File $image The image file to be processed
+     * @param string $path The path to save the processed image file
+     * @param int $coverType The type to process the image to
```

Comment:
`$coverType` is an array type, judging by line 136, not an int. The argument itself on line 135 should be typed as such, too.